docs/references/strategies/languages/dotnet/nuget-qr.md

```diff
@@ -0,0 +1,17 @@
+# Quick reference: NuGet
```

Comment:
This file is new in this PR and not referenced anywhere in the documentation. There are other `<language>-qr.md` files in the repo, but I think it's confusing to have both `nuget.md` and `nuget-qr.md`. 

So, I know you didn't add it, but I think we should delete it